flowcraft/generator/engine.py

```diff
@@ -101,7 +101,8 @@
         "split_assembly": meta.SplitAssembly,
         "trimmomatic": readsqc.Trimmomatic,
         "true_coverage": readsqc.TrueCoverage,
-        "viral_assembly": assembly.ViralAssembly
+        "viral_assembly": assembly.ViralAssembly,
+        "abyss": assembly.Abyss
```

Comment:
Although this will probably be refactored according to #145 , can you keep the alphabetical order for now?

Reply:
Done